lib/utils/extension/build_context_extension.dart

```diff
@@ -186,7 +189,7 @@ extension ContextExtensionss on BuildContext {
     Navigator.of(this).popUntil((route) => route.isFirst);
   }
 
-  void goToRoomWithEvent(MatrixEvent event) {
-    go('/rooms/${event.roomId}?event=${event.eventId}');
+  void goToRoomWithEvent(String roomId, String eventId) {
```

Comment:
why not keep `event` because maybe in the future we need more information of event?

Reply:
but harder to reuse, IMO, keep only necessary information